tests/sentry/api/bases/test_integration.py

```diff
@@ -7,8 +7,7 @@
 
 
 class IntegrationEndpointTest(APITestCase):
-    def setUp(self):
-        self.endpoint = IntegrationEndpoint()
+    endpoint = IntegrationEndpoint()
```

Comment:
This seems weird, why are we declaring this at the class level? Tbh, it probably makes sense to rename this since it's colliding with the `endpoint` property.

Reply:
actually on second thought, I bet this isn't even using the facilities of `APITestCase`